src/app/examples/modal-example/modal-example.component.html

```diff
@@ -0,0 +1,28 @@
+<p>
+  <kirby-button theme="dark" label="Open dark modal" (click)="openDarkModal()"></kirby-button>
+</p>
+<p>
+  <kirby-button theme="cookbook" label="Open light modal" (click)="openLightModal()"></kirby-button>
+</p> 
+
+<kirby-modal theme="dark" (closingModal)='closeDarkModal()' [isOpen]='isDarkModalOpen' size="large">
```

Comment:
Har vi undersøgt muligtheden for at lave vores modal som en service istedet? Kunne være fedt, hvis den kunne åbnes fra controlleren, så vi ikke har vores modals inlinet i DOM'en.